lib/kitto.ex

```diff
@@ -85,6 +86,7 @@ defmodule Kitto do
   defp children(_env) do
     [supervisor(__MODULE__, [], function: :start_server),
      supervisor(Kitto.Notifier, []),
+     worker(Kitto.Hooks, []),
```

Comment:
Since it doesn't depend on any of the other children, i think we should start it last.